src/js/common/schemaform/review/PhoneNumberWidget.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+// When coming from a PhoneNumberWidget (not review), value will be only numbers
+export default function PhoneNumberWidget({ value }) {
+  let formatted = value;
+  if (value && value.length === 10) {
+    formatted = `(${value.substr(0, 3)}) ${value.substr(3, 3)}-${value.substr(6)}`;
+  }
```

Comment:
There are two ways I can think of doing it. We could add a transform option to the uiSchema. The other way is a higher order component: a function that takes a transform function as a parameter and returns a component that uses it.

That said, we have two instances of this so far. I'm hesitant to add harder to understand code for a use case that I'm not sure goes beyond these two fields.